app/views/users/index.html.erb

```diff
@@ -1,2 +1,16 @@
-<h1> Here is list of users and their corresponding number of posts
-</h1>
+<div class="users-index-wrapper">
+<h1 class="header-text">Here is list of users and their corresponding number of posts</h1>
```

Comment:
- [ ] Please delelte this line to match the given layout.

Reply:
line removed